Receive/includes/bms-rx-data.h

```diff
@@ -0,0 +1,87 @@
+/************************ Includes **************************/
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+#ifdef _WIN32
+#include <Windows.h>
+#else
+#include <unistd.h>
+#endif
+
+
+
+/************************ Defines **************************/
+
+#define TEMPERATURE_MIN     20.0
+#define TEMPERATURE_MAX     80.0
+#define CHARGERATE_MIN 0.0
+#define CHARGERATE_MAX 5.0
+
+#define SMA_RANGE 5
+#define MAXLENGTH 200
+
+#define VALUE_NOTFOUND 65532
+#define INVALID_SMARANGE 65530
+#define INVALID_VALUE 65535
+
+
+
+
+/************************ Variables **************************/
+enum BmsParameters_e{
+    temperature = 0,
+    charge_rate,
+    NumberOfParameters
+};
+
+typedef struct BatteryParameters{
+    char ParameterName[20];
+    float minValue;
+    float maxValue;
+}BatParams_s;
+
+typedef struct BatteryParametersOut{
```

Comment:
`BatteryParametersOut` is about statistics, while `BatteryParameters` is the input data. You have used the same word `Parameters` to mean two different things. Avoid this ambiguity